scripts/deployment_manager/deployment_manager.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+"""
+ Copyright (c) 2018-2021 Intel Corporation
```

Comment:
please, add install rules for this tool to `scripts/CMakeLists.txt`

Reply:
Do we have tickets to add open source tests for DM?

Reply:
Nope, I'll create

Reply:
>please, add install rules for this tool to scripts/CMakeLists.txt
Done